cmd/ncproxy/run.go

```diff
@@ -159,6 +162,29 @@ func run() error {
 		}
 	}
 
+	// setup ncproxy databases
+	if *dbPath == "" {
+		// default location for ncproxy database
+		binLocation, err := os.Executable()
+		if err != nil {
+			return err
+		}
+		*dbPath = filepath.Dir(binLocation) + "networkproxy.db"
+	} else {
+		// If a db path was provided, make sure parent directories exist
+		dir := filepath.Dir(*dbPath)
+		if _, err := os.Stat(dir); err != nil {
+			if err := os.MkdirAll(dir, 0); err != nil {
+				return errors.Wrap(err, "failed to make database directory")
+			}
+		}
+	}
+
+	db, err := bolt.Open(*dbPath, 0600, nil)
```

Comment:
Might make more sense to do the db open inside `newComputeAgentStore`, we could just put `dbPath` into the config that is stored in the server object.

Reply:
Alternately if we expect to have other state to track in the future, should the db be stored in `server` and used to track all persistent state.

Reply:
I'll make those changes.